git_machete/git_operations.py

```diff
@@ -793,13 +815,20 @@ def check_hook_executable(self, hook_path: str) -> bool:
             advice_ignored_hook = self.get_config_attr_or_none("advice.ignoredHook")
             if advice_ignored_hook != 'false':  # both empty and "true" is okay
                 # The [33m color must be used to keep consistent with how git colors this advice for its built-in hooks.
-                print(colored(f"hint: The '{hook_path}' hook was ignored because it's not set as executable.", AnsiEscapeCodes.YELLOW), file=sys.stderr)
-                print(colored("hint: You can disable this warning with `git config advice.ignoredHook false`.", AnsiEscapeCodes.YELLOW), file=sys.stderr)
+                print(colored(f"hint: The '{hook_path}' hook was ignored because it's not set as executable.",
+                              AnsiEscapeCodes.YELLOW),
+                      file=sys.stderr)
+                print(colored("hint: You can disable this warning with `git config advice.ignoredHook false`.",
+                              AnsiEscapeCodes.YELLOW),
+                      file=sys.stderr)
             return False
         else:
             return True
 
-    def merge(self, branch: LocalBranchShortName, into: LocalBranchShortName, opt_no_edit_merge: bool) -> None:  # refs/heads/ prefix is assumed for 'branch'
+    def merge(self, branch: LocalBranchShortName,
+              into: LocalBranchShortName,
+              opt_no_edit_merge: bool
+              ) -> None:  # refs/heads/ prefix is assumed for 'branch'
```

Comment:
This is now self-evident from `branch` and `into` param types, you can remove the comment while you're here